tools/cli/go-whisk-cli/commands/action.go

```diff
@@ -544,6 +544,9 @@ func parseAction(cmd *cobra.Command, args []string) (*whisk.Action, bool, error)
       action.Exec.Kind = "swift"
     } else if ext == ".js" {
       action.Exec.Kind = "nodejs:6"
+    } else if ext == ".zip" {
```

Comment:
The .zip file extension is pretty general purpose.  I think having a hard association with .zip and nodejs:6 is to confining.  Perhaps if .zip is the extension, the `kind` flag must be specified.